packages/web3-eth/package.json

```diff
@@ -31,7 +31,8 @@
         "web3-eth-personal": "1.0.0-beta.52",
         "web3-net": "1.0.0-beta.52",
         "web3-providers": "1.0.0-beta.52",
-        "web3-utils": "1.0.0-beta.52"
+        "web3-utils": "1.0.0-beta.52",
+        "eth-lib": "0.2.8"
```

Comment:
Why do you add a dependency here of eth-lib you can use the sha3 method on web3-utils and not being in that dependency.

Reply:
I agree, I changed it to use `this.utils` instead.